types/session_pool/session_pool.d.ts

```diff
@@ -70,13 +70,11 @@ export class SessionPool extends EventEmitter {
     constructor(options?: SessionPoolOptions | undefined);
     maxPoolSize: number;
     createSessionFunction: any;
-    /** @type {SessionPoolOptions} */
-    sessionOptions: SessionPoolOptions;
+    sessionOptions: {};
```

Comment:
this won't work in TS, any property you try to access in strict mode will fail without a typecast `as unknown as any`

Reply:
Yeah, I expected something like that, but as I said above, JS code readability and docs are more important.

Let's figure out a different way to do this.